docs/samples/deployments/spark/presidio_anonymize_blobs.py

```diff
@@ -0,0 +1,153 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Anonymize PII Entities in text files
+# MAGIC
+# MAGIC <br>Using Presidio, anonymize PII content of files in an Azure Storage account.
+# MAGIC
+# MAGIC <br>The following code sample will:
+# MAGIC <ol>
+# MAGIC <li>Import the content of text files located in an Azure Storage blob folder</li> # noqa D501
+# MAGIC <li>Anonymize the content using Presidio</li>
+# MAGIC <li>Write the anonymized content back to the Azure Storage blob account</li>
+# MAGIC </ol>
+
+# COMMAND ----------
+
+from azure.storage.blob import BlobServiceClient
+from presidio_analyzer import AnalyzerEngine
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import AnonymizerConfig
+from pyspark.sql.types import StringType
+from pyspark.sql.functions import input_file_name, regexp_replace
+from pyspark.sql.functions import col, pandas_udf
+import pandas as pd
+
+dbutils.widgets.text(
+    "storage_account_name", "", "Blob Storage Account Name"
+)  # noqa: F821
+dbutils.widgets.text("storage_container_name", "", "Blob Container Name")  # noqa: F821
+dbutils.widgets.text(
+    "storage_account_access_key", "", "Storage Account Access Key"
+)  # noqa: F821
+dbutils.widgets.text(
+    "storage_input_folder", "input", "Storage Account Access Key"
```

Comment:
Perhaps you intended to have "Storage input folder" as text instead of "Storage Account Access Key" ?